MAINTAINERS.md

```diff
@@ -0,0 +1,36 @@
+<!-- This is a generated file. Please do not edit directly -->
+
+# Maintainers
+
+This file lists how the ChefDK project is maintained. When making changes to the system, this
+file tells you who needs to review your patch - you need a simple majority of maintainers
+for the relevant subsystems to provide a :+1: on your pull request. Additionally, you need
+to not receive a veto from a Lieutenant or the Project Lead. ChefDK also has an internal Clients
+Pool team of all members have maintainership authority.
```

Comment:
Is there any reason not to add Clients POOL members to this list under a separate heading? I think it's important that they also be added to `@chef/chef-dk-maintainers`.